src/solver/conv_winoRxS_ultra_f2x3.cpp

```diff
@@ -43,7 +43,19 @@
 MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_WINOGRAD_ULTRA_RXS_F2X3)
 MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_AMD_WINOGRAD_ULTRA_RXS_F2X3_PERF_VALS)
 
-#define MAX_CU_LIMIT 512
+static inline auto IsPowerOf2(size_t v) { return ((v - 1) & v) == 0; }
+
+static inline auto GetBestInterleaveParam(const int N, const int C, const int H, const int W)
+{
+    int best_intl_factor = 1;
+
+    if(IsPowerOf2(W) && (N * C * H * W >= std::pow(2, 25)))
```

Comment:
I don't think it should depends on N. Also `IsPowerOf2(W)` looks to strict - I imagine any multiple of some power of 2 should fit better.